utils/bash/Makefile

```diff
@@ -43,6 +43,7 @@ endef
 define Package/bash/conffiles
 /etc/bash.bashrc
 /etc/bash.bash_logout
+/etc/profile.d/sys_bashrc.sh
```

Comment:
Normally, conf.d files are prefixed with a number to allow the user to add files before or after existing conf files. Something like 50-sys_bashrc.sh